docs/deployment/plugins/k8s/index.rst

```diff
@@ -610,23 +427,229 @@ Specify plugin configuration
    
       .. tabs:: 
 
+        .. group-tab:: Demo (sandbox) cluster
+
+           Enable the Spark plugin on the demo cluster by adding the following block to ~/.flyte/sandbox/config.yaml:
+
+        .. code-block:: yaml
```

Comment:
needs to be indented. i think you haven't copied the whole config.

Reply:
I just checked and there are the same three sections in the live docs: `tasks`, `plugins` and `cluster_resources`. Also checked indentation and it should be fine. It uses the same spacing as in a working deployment

Reply:
i meant https://github.com/flyteorg/flyte/pull/4670/files/41cba8272c6fb7fe34d0fcb02bbf39c1774fc038#diff-5e363febf4c9f8a60b6f884277d82eb90f1d76e1f2a5b14037133c1c9ff36adeL154-L215 section.

Reply:
i haven't tested the spark one 'cause it didn't work for me earlier. but i believe this is how it needs to be done. but we need to test the config.